### DISCUSSION TITLE
Welcome to PHP-FFmpeg-video-streaming Discussions!

### DISCUSSION THREAD

**Discussion by aminyazdanpanah** [aminyazdanpanah/PHP-FFmpeg-video-streaming#97] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by talasokratas** [comment]
It seems more convenient and more up-to-date way to generate HLS streams than your SHAKA php wrapper, which I already use and it's great for generating m3u8 playlists.  I wander why there's no way to add multi language audio files using PHP-FFmpeg-video-streaming, it could be similar like adding subtitle tracks with some HLSAudio class

**Reply by aminyazdanpanah** [reply]
Unfortunately, I am too busy these days and can not spend time on this library but you are right. It should be easy to add this feature.
